requirements/docs.txt

```diff
@@ -21,3 +21,6 @@ lightning-utilities >=0.11.0, <0.12.0
 
 # installed from S3 location and fetched in advance
 lai-sphinx-theme
+
+# needed for replacing stable keyword with latest version linkd
+tensorboard
```

Comment:
I don't understand why the package would be needed to manipulate the links? Can we try to avoid it, or ignore the tensorboard links?

Reply:
it load version from local env until this lands https://github.com/Lightning-AI/utilities/pull/243

Reply:
so this is a resolution on actual state that PL dos can't be build :thinking:

Reply:
```suggestion
tensorboard  # todo: remove this when updating Utils to 0.11.1+
```

Reply:
@Borda can you open a PR for this?

Reply:
it is a bit tricky to make a new PR on change that was not yet merged :)

Reply:
fixing it in #19697